notario/normal.py

```diff
@@ -1,10 +1,12 @@
+import copy
 from notario.utils import ndict, is_nested_tuple
 
 
 class BaseNormalize(object):
 
     def __init__(self, data, schema):
-        self.raw_data = data
+        copied_data = copy.deepcopy(data)
+        self.raw_data = copied_data
```

Comment:
it doesn't seem like `copied_data` is used anywhere else, do you think it might be fine to just assign it directly to `self.raw_data` ?